tools/rapids-conda-retry

```diff
@@ -119,6 +119,9 @@ function runConda {
         elif grep -q "Timeout was reached" "${outfile}"; then
             retryingMsg="Retrying, found 'Timeout was reached' in output..."
             needToRetry=1
+        elif grep -q "Unexpected error [0-9]* on netlink descriptor [0-9]*" "${outfile}"; then
```

Comment:
Feel free to reject this, as I haven't seen lots of these errors myself. If you expect that there should always be a number, then + is better than *.

```suggestion
        elif grep -q "Unexpected error [0-9]+ on netlink descriptor [0-9]+" "${outfile}"; then
```

Reply:
I have always seen a number there, I think the `+` is a good suggestion.